devdox/generate_token.py

```
@@ -1,5 +1,5 @@
 from clerk_backend_api import Clerk
-from app.config import settings
+from devdox.app import settings
```

Pylint (3.3.7)

[error] No name 'settings' in module 'devdox.app'

(E0611)